### PR TITLE
[2488] Add more configurable ENV's for s3 ActiveStorage

### LINKED ISSUE #2488: add ENV's to s3 config
add S3_ACCESS_KEY_ID and S3_SECRET_ACCESS_KEY_ID to be configurable as ENV's

### PR DESCRIPTION
**Pull request by kmarszalek**
Add `S3_ACCESS_KEY_ID` and `S3_SECRET_ACCESS_KEY_ID` as
configurable ENV's for S3 ACtiveStorage.

Closes #2488

### CODE REVIEW

File: config/storage.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ local:
 s3:
   service: S3
   endpoint: <%= ENV["S3_ENDPOINT"] || "https://s3.cloud.cyfronet.pl/" %>
-  access_key_id: <%= Rails.application.credentials.dig(:s3, :access_key_id) %>
-  secret_access_key: <%= Rails.application.credentials.dig(:s3, :secret_access_key) %>
+  access_key_id: <%= ENV["S3_ACCESS_KEY_ID"] || Rails.application.credentials.dig(:s3, :access_key_id) %>
+  secret_access_key: <%= ENV["S3_SECRET_ACCESS_KEY_ID"] || Rails.application.credentials.dig(:s3, :secret_access_key) %>
```
Comment (jswk) [inline config/storage.yml:13]: Why not just `S3_SECRET_ACCESS_KEY` (without the trailing `_ID`)?

File: README.md [hunk 2]
```diff
@@ -200,6 +200,8 @@ We are currently using the following ENV variables:
   * `S3_STORAGE` - set true to change ActiveStorage to S3
   * `S3_BUCKET` - active storage S3 bucket
   * `S3_ENDPOINT` - active storage S3 endpoint
+  * `S3_ACCESS_KEY_ID` - active storage S3 access key
+  * `S3_SECRET_ACCESS_KEY_ID` -active storage S3 secret access key
```
Comment (jswk) [inline README.md:204]: Space after `-`.

**Review by jswk — COMMENTED** [review]
Shouldn't this process of setting the new envs be described in `docs/upgrade/3.30.0.md`?
At least some changes to envs setting are described in the upgrade docs, it would be nice to have this documented that way too.

**Review by wziajka — APPROVED** [review]

**Review by jswk — CHANGES_REQUESTED** [review]
Also correct the commit message to match the corrected env name and the upgrade MD file.

**Pull request merged**